changes.xml

```diff
@@ -24,6 +24,9 @@
   <body>
 
     <release version="1.6.0" date="not released">
+      <action type="add" dev="nbellack">
+        Role aem-dispatcher: Enable HTTP/2 by default.
```

Comment:
make clear that HTTP/2 support was added with this release (an it's enabled by default).
as it's written currently it sounds that support was available already and is now only enabled by default.

Reply:
Changed in latest commit